calico-cloud/image-assurance/understanding-scan-results.mdx

```diff
@@ -59,6 +59,14 @@ To change the CVSS threshold values, note the following:
 - Changes to threshold values take effect immediately and alter the scan results for images already in the system
 - If you are using admission controller policies, changing a value may allow pods in your Kubernetes cluster that were previously being blocked, to now be deployed or vice versa.
 
+## Exploit Data
+
+The scanning process also attaches EPSS and known exploit data onto each image and vulnerability.
+
+EPSS information can be found [here](https://www.first.org/epss/model).
```

Comment:
Please give your anchor a descriptive text. For example: For information about EPSS, see [Title of Interesting Article](link).

Reply:
Updated